source/dagger/gameplay/plight/plight.cpp

```diff
@@ -80,7 +91,8 @@ void Plight::GameplaySystemsSetup(Engine &engine_)
 {
     engine_.AddSystem<TopdownControllerSystem>();
     engine_.AddSystem<PlightCollisionsSystem>();
-    engine_.AddSystem<PlightCombatSystem>();
+    //engine_.AddSystem<PlightCombatSystem>();
```

Comment:
are u sure u need to comment it out?

Reply:
No need , I just commented it out so that I can test the new movement and aiming without draining stamina while running. It works fine with the combat system as well , I just forgot to remove the comment , will do !